frb/scripts/pz_dm.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+"""
+Estimate p(z|DM) for an assumed location on the sky and DM_FRB
+"""
+from IPython import embed
+
+def parser(options=None):
+    import argparse
+    # Parse
+    parser = argparse.ArgumentParser(description='Script to print a summary of an FRB to the screen [v1.0]')
+    parser.add_argument("coord", type=str, help="Coordinates, e.g. J081240.7+320809 or 122.223,-23.2322 or 07:45:00.47,34:17:31.1 or FRB name (FRB180924)")
+    parser.add_argument("DM_FRB", type=float, help="FRB DM (pc/cm^3)")
+    parser.add_argument("--dm_hostmw", type=float, default=100., help="Assumed DM contribution from MW and Host")
+    parser.add_argument("--cl", type=tuple, default=(2.5,97.5), 
+                        help="Confidence limits for the z estimate [default is a 95 percent c.l., (2.5,97.5)]")
+
+    if options is None:
+        pargs = parser.parse_args()
+    else:
+        pargs = parser.parse_args(options)
+    return pargs
+
+
+def main(pargs):
+    """ Run
+    """
+    import json
+    import os
+    import numpy as np
+    from pkg_resources import resource_filename
+
+    from linetools import utils as ltu
+    from linetools.scripts.utils import coord_arg_to_coord
+
+    from frb.galaxies import nebular
+    from frb.galaxies import photom
+    from frb.galaxies import utils as frb_gal_u
+    from frb import mw
+    from frb.dm import prob_dmz
+
+    # Deal with coord
+    icoord = ltu.radec_to_coord(coord_arg_to_coord(pargs.coord))
+
+    # NE 2001
+    DM_ISM = mw.ismDM(icoord)
+    print("")
+    print("-----------------------------------------------------")
+    print(f"NE2001 = {DM_ISM:.2f}")
+
+    # DM Cosmic
+    DM_cosmic = pargs.DM_FRB - DM_ISM.value - pargs.dm_hostmw
```

Comment:
If DM_ISM is being subtracted here, maybe clarify in the helptext for `--dm_hostmw` that they only need to specify the halo contribution for MW. I can see someone including the NE2001 estimate in that value otherwise.

Reply:
good call.  Have updated the script and docs.